build-conflicts.php

```diff
@@ -34,6 +34,11 @@
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromGithubApi;
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromMultipleSources;
 
+use function array_keys;
+use function count;
+use function file_get_contents;
+use function in_array;
+use function json_decode;
```

Comment:
```suggestion
use Psl\Json;
```

$result = Json\decode($json);

or better:

```php
use Psl\Json;
use Psl\Type;

$result = Json\typed($json, Type\shape([
  'foo' => Type\string(),
  'bar' => Type\vec(Type\int()), // vec<T> is list<T>
  'baz' => Type\dict(Type\int(), Type\string()), // dict<Tk, Tv> is array<Tk, Tv>
]);

```